src/corner_kick/src/ros/RosConnector.ts

```diff
@@ -0,0 +1,137 @@
+import ROSLib from 'roslib';
+
+const ROS_URL = 'ws:localhost:9090'; // TODO: Make this parameterized
```

Comment:
could probably go in constants file? On the other hand, it would be really nice if we could set at least the port at the roslaunch level so that it could match with the rosbridge port set in the same file.

Reply:
Moved to the constant file